src/e_lgamma.c

```diff
@@ -21,16 +21,19 @@
  * Method: call __ieee754_lgamma_r
  */
 
-#include <openlibm_math.h>
+#include <float.h>
 
+#include <openlibm_math.h>
 #include "math_private.h"
 
-DLLEXPORT double
+extern int signgam;
+
+double
 __ieee754_lgamma(double x)
 {
-#ifdef OPENLIBM_ONLY_THREAD_SAFE
```

Comment:
I still think that we need the `extern int signgam` if not in the THREAD_SAFE compile mode. All these patches should perhaps be external patches, so that we can sync with msun upstream and then apply our patches.